README.md

```diff
@@ -266,6 +273,11 @@ This specification proposes Currency Specific Parts for USDC and BTC.
 ###### Format
 `urn:btc:spaddr:<address>`
 
+##### DNS (BIP-353)
+
+###### Format
+`urn:btc:dns:<internet-identifier>`
```

Comment:
I thought `dns` was the most appropriate prefix here, as the Money Address is resolved via a TXT lookup.

Reply:
@hugomd do you know if there are any other dns based address/invoice/offer lookup BIPs or conventions that are used within Bitcoin? if not, `dns` makes sense to me!